docs/en/tutorials/cdc_dfu_flash.rst

```diff
@@ -66,7 +66,7 @@ Go to the Tools menu in the Arduino IDE and set the following options:
 
 * USB DFU On Boot -> Enabled
 
-Setp 3 - Flash
+Step 3 - Flash
```

Comment:
Thank you of fixing the typo. 
May I suggest to align this step to match other steps format like this: **3. Flash**
```suggestion
3. Flash
```